### PR TITLE
Add empty page Technologie

### PR DESCRIPTION
**Pull request by Morel-Victor**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -41,6 +42,9 @@ export default function App() {
         <Route path="/" element={<Home />}>
           Home
         </Route>
+        <Route path="/technologie" element={<Technologie />}>
```
Comment (Caroline-CD) [inline src/App.jsx:45]: <Route path="/technologie" element={<Technologie />} />

Easier to write it like that I think ! 
Can you remove "zdzq" ?

**Review by Caroline-CD — COMMENTED** [review]
Little modifications

**Review by Caroline-CD — APPROVED** [review]
Ok :100:  for me

**Review by Caroline-CD — APPROVED** [review]
OK

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**